pyccel/ast/cudaext.py

```diff
@@ -48,22 +48,12 @@
 )
 
 #==============================================================================
-class CudaNewArray(PyccelInternalFunction):
+class CudaNewArray(NumpyNewArray):
     """ Class from which all Cuda functions which imply a call to Allocate
     inherit
     """
     __slots__ = ()
-    #--------------------------------------------------------------------------
-    @staticmethod
-    def _process_order(rank, order):
-
-        if rank < 2:
-            return None
-
-        order = str(order).strip('\'"')
-        if order not in ('C', 'F'):
-            raise ValueError('unrecognized order = {}'.format(order))
-        return order
+    pass
```

Comment:
`pass` is not necessary. It is only needed when there is no code in the indented block.
E.g.
```python
class CudaNewArray(NumpyNewArray):

class A:
    ....
```
is problematic and needs `pass`, but:
```python
class CudaNewArray(NumpyNewArray):
     """ My docstring
     """

class A:
    ....
```
or:
```python
class CudaNewArray(NumpyNewArray):
    __slots__ = ()

class A:
    ....
```
is fine